### ISSUE TITLE
Js functions present in Js file located in wwwroot of WebApplication not working

### ISSUE THREAD

**Issue by alberspim** [AngleSharp/AngleSharp.Js#74]
First of all, keep up the good work! I love AngleSharp and it is a great addition for my tests.

I need some help how to use external script resources. (Using AngleSharp.Js version 0.14.0)

I have a small .netCore MVC web project and want to get a page that makes use of javascript functions that are present in a .js file inside the wwwroot folder of the web project.

Plane.cshtml

![image](https://user-images.githubusercontent.com/12752414/107622340-0151ec00-6c58-11eb-9ee4-006d2c99134d.png)

plane.js

![image](https://user-images.githubusercontent.com/12752414/107622521-42e29700-6c58-11eb-8ac0-ad40ba9ee6e3.png)

I have created a helper to retrieve the HtmlDocument

![image](https://user-images.githubusercontent.com/12752414/107622835-bab0c180-6c58-11eb-9a63-c6704f0d5f10.png)

Long story short, the controller method "plane/GetPlaneData" to retrieve the data is not called.
Is this even possible? If yes, could you give some tips on how to get this to work?

Many thanks
